mail_composer_cc_bcc/__manifest__.py

```diff
@@ -3,12 +3,12 @@
 {
     "name": "Email CC and BCC",
     "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.0",
     "development_status": "Alpha",
     "category": "Social",
     "website": "https://github.com/OCA/social",
     "author": "Camptocamp SA, Odoo Community Association (OCA)",
-    "maintainers": ["hailangvn2023"],
+    "maintainers": ["hailangvn2023", "trisdoan"],
```

Comment:
you can remove Hai from here

Reply:
in the final PR description, you can add something like: "My colleague Hai is busy on other topics, I will take over as a maintainer of this module from now on."